### PR TITLE
Remove (page).URL from our layouts as it is deprecated

### PR DESCRIPTION
**Pull request by MarkvanMents**
Files are slightly different lengths, but cannot identify any differences.
Output compared in VS Code - no differences found
Formatted JSON in VS Code and compared first 150000 characters (cannot format entire file) - no differences found

### CODE REVIEW

**Comment by MarkvanMents** [comment]
Also removed the else in the navbar.html as this would also trigger a deprecation - cannot identify any issues.

File: layouts/_default/list.mxdocsalgolia.json [hunk 1]
```diff
@@ -149,7 +149,7 @@ Define Global variables
                         {{- end -}}
                         {{- $scratch.SetInMap $uniqueHit "weight.tag_name" $tagName -}}
                         {{- /* .URL takes into account any changes by using the url: tag in the front matter */ -}}
```
Comment (NatasaKralj) [inline layouts/_default/list.mxdocsalgolia.json]: I would only change the comment on line 151 to reflect the new parameter being used. While `.RelPermalink` does give the same result as `.Params.url` (grabs the front matter parameter), it is a parameter that can be configured separately through `permalink`.
"Grabs address relative to baseURL, can be affected by permalink parameter" as the comment perhaps?

**Comment by NatasaKralj** [comment]
The changes look good, work well.

File: layouts/_default/list.mxdocsalgolia.json (hunk 1 above)
Comment (MarkvanMents) [reply layouts/_default/list.mxdocsalgolia.json]: Good idea - I'll do that, then merge.

**Pull request merged**